pkg/manifests/manifests.go

```diff
@@ -268,6 +270,25 @@ func newIngressControllerDeployment(conf *config.Config) *appsv1.Deployment {
 	}
 }
 
+func newIngressControllerConfigmap(conf *config.Config) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      IngressControllerName,
+			Namespace: conf.NS,
+			Labels:    topLevelLabels,
+		},
+		Data: map[string]string{
+			// Can't use 'allow-snippet-annotations=false' to reduce injection risk, since we require snippet functionality for OSM routing.
+			// But we can still protect against leaked service account tokens.
+			"annotation-value-word-blocklist": "load_module,lua_package,_by_lua,location,root,proxy_pass,serviceaccount,{,},'",
```

Comment:
Where did this list come from? Might be useful to add a reference to the source as a comment